fact-bounty-flask/migrations/versions/02db7a3aaa95_.py

```diff
@@ -0,0 +1,66 @@
+"""empty message
```

Comment:
I think that two migrations files are unnecessary as only a new column is added to user table, so try to generate only corresponding to that change